pkg/agent/cluster.go

```diff
@@ -466,6 +497,22 @@ func humanFriendlyClusterInstallStatus(status string) string {
 		models.ClusterStatusPreparingForInstallation:    "Preparing cluster for installation",
 		models.ClusterStatusReady:                       "Cluster is ready for install",
 	}
-	return clusterStoppedInstallingStates[status]
-
+	hostStoppedInstallingStates := map[string]string{
+		models.ClusterStatusAddingHosts:                 "Cluster is adding host",
+		models.ClusterStatusCancelled:                   "Host installation cancelled",
+		models.ClusterStatusError:                       "Host has error(s)",
+		models.ClusterStatusFinalizing:                  "Finalizing host installation",
+		models.ClusterStatusInstalling:                  "Host installation in progress",
+		models.ClusterStatusInstallingPendingUserAction: "Host installation started but now requires user input",
+		models.ClusterStatusInsufficient:                "Host is not ready for install. Check validations",
+		models.ClusterStatusPendingForInput:             "User input is required to continue host installation",
+		models.ClusterStatusPreparingForInstallation:    "Preparing host for installation",
+		models.ClusterStatusReady:                       "Host is ready for install",
+	}
```

Comment:
I had the wrong impression that the cluster status will be logged when the host start installing, so that was why I added the translations for hosts. But after testing it a bit more, the only one that prints out is ClusterStatusAddingHosts which doesn't need translation. So yes, the translations I had can be removed.